### DISCUSSION TITLE
Single category display

### DISCUSSION THREAD

**Discussion by ersingo** [bootscore/bootscore#612] [Q&A]
Hello;

I have a question for bootscore_category_badge function.

I add the post to more than one category. But I want it to show the last category on the home page according to the hierarchy or the category I set as default. how do I do this? I use Yoast seo. with this I set the primary category. but I have no idea how to do this.

![Screenshot_5](https://github.com/bootscore/bootscore/assets/32128295/2b357213-3398-42f1-86e2-bdc86e7c1bf5)

In short, what I want is this; Even if I have marked 4 categories, I want to get a single category output. the category I set as primary or the last category according to the hierarchy.

**Comment by crftwrk** [comment]
You can‘t do it through the `bootscore_category_badge` function, because this shows all selected categories. 

But you can tell Yoast to show only primary category. Quickly googled it and found:  

- https://joshuawinn.com/using-yoasts-primary-category-in-wordpress-theme/
- https://www.tannerrecord.com/how-to-get-yoasts-primary-category/

Not tested, but this seems to be the way to solve it.
